packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardPreview.styles.ts

```diff
@@ -0,0 +1,83 @@
+import { getGlobalClassNames, FontSizes } from '../../Styling';
+import { IDocumentCardPreviewStyleProps, IDocumentCardPreviewStyles } from './DocumentCardPreview.types';
+
+export const GlobalClassNames = {
+  root: 'ms-DocumentCardPreview',
+  icon: 'ms-DocumentCardPreview-icon',
+  iconContainer: 'ms-DocumentCardPreview-iconContainer',
+  previewFileTypeIcon: 'ms-DocumentCardPreview-icon'
+};
+
+export const getStyles = (props: IDocumentCardPreviewStyleProps): IDocumentCardPreviewStyles => {
+  const { theme, className, isFileList } = props;
+  const { palette } = theme;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  return {
+    root: [
+      classNames.root,
+      {
+        borderBottom: `1px solid ${palette.neutralLight}`,
+        position: 'relative',
+        backgroundColor: isFileList ? palette.white : palette.neutralLighterAlt,
+        overflow: `hidden`
+      },
+      className
+    ],
+    previewIcon: [
+      classNames.iconContainer,
+      {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100%'
+      }
+    ],
+    previewFileTypeIcon: [
+      classNames.previewFileTypeIcon,
+      {
+        left: '10px',
+        bottom: '10px',
+        position: 'absolute'
+      }
+    ],
+    fileList: {
+      padding: '16px 16px 0 16px',
+      listStyleType: 'none',
+      margin: 0,
+      selectors: {
+        li: {
+          height: '16px',
+          lineHeight: '16px',
+          marginBottom: '8px',
```

Comment:
nit: again not a blocker but all these values can be just numbers. No need to specify `px` everywhere. Same goes for all the `.styles` files.